AdminServer/appscale/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
I like to see something what I didn't know about python=)
It forces me to google and learn more.
But... why do you use `__import__`? [Documentation](https://docs.python.org/2/library/functions.html#__import__) says that it should be used only in case you want to import a module whose name is only known at runtime.

I'd suggest to replace it with
```
import pkg_resources
pkg_resources.declare_namespace(__name__)
```